mne/utils/check.py

```diff
@@ -330,14 +330,29 @@ def _soft_import(name, purpose, strict=True):
         functionality the package provides to MNE-Python.
     strict : bool
         Whether to raise an error if module import fails.
+    pip_name : str
+        The PyPI package name if it does not match the module name.
```

Comment:
```suggestion
```
It's not a passed parameter, it's hard-coded into the function. Therefore shouldn't be in the docstring